--- mate-file-manager-gksu-1.6.0/configure.ac~	2013-05-12 00:47:40.689730232 +0300
+++ mate-file-manager-gksu-1.6.0/configure.ac	2013-05-12 00:47:19.382465408 +0300
@@ -6,7 +6,7 @@
 AC_INIT(caja-gksu, 1.6.0, http://www.mate-desktop.org/)
 AM_INIT_AUTOMAKE(AC_PACKAGE_NAME, AC_PACKAGE_VERSION)
 AC_CONFIG_SRCDIR(caja-gksu/libcaja-gksu.c)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 AM_MAINTAINER_MODE
 
